will_kitchell/package.json

```diff
@@ -16,18 +16,39 @@
   },
   "author": "william.kitchell@gmail.com",
   "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.14.1",
+    "express": "^4.13.3",
+    "mongoose": "^4.2.5"
+  },
   "bugs": {
     "url": "https://github.com/picklekitch/mongo_backed_rest_api/issues"
   },
   "homepage": "https://github.com/picklekitch/mongo_backed_rest_api#readme",
   "devDependencies": {
     "angular": "^1.4.8",
+    "angular-mocks": "^1.4.8",
     "bcrypt": "^0.8.5",
     "body-parser": "^1.14.1",
+    "chai": "^3.4.1",
+    "chai-http": "^1.0.0",
     "eat": "^0.1.1",
     "express": "^4.13.3",
     "gulp": "^3.9.0",
+    "gulp-concat-css": "^2.2.0",
+    "gulp-jshint": "^1.12.0",
+    "gulp-minify-css": "^1.2.2",
+    "gulp-mocha": "^2.1.3",
+    "gulp-sass": "^2.1.1",
+    "gulp-sourcemaps": "^1.6.0",
+    "jshint": "^2.8.0",
+    "jasmine-core": "^2.3.4",
+    "karma": "^0.13.15",
+    "karma-jasmine": "^0.3.6",
+    "karma-phantomjs-launcher": "^0.2.1",
+    "mocha": "^2.3.3",
+    "phantomjs": "^1.9.19",
     "mongoose": "^4.2.9",
-    "webpack-stream": "^2.3.0"
+    "webpack-stream": "^2.2.0"
```

Comment:
You have dependencies also listed under devDependencies, and there are other devDependencies that should be in dependencies.